### ISSUE TITLE
Docs are scattered; Template files are inconsistent. 

### ISSUE THREAD

**Issue by KalobMTaulien** [pinax/pinax-stripe#556]
#### Issue Summary

Installing `pinax-stripe` was easy enough, but customizing the templates has been a huge time vampire. They don't come in this repo so I had to do a search and found [your other theme repo](https://github.com/pinax/pinax-theme-bootstrap/tree/master/pinax_theme_bootstrap/templates/pinax/stripe). Downloaded the template files and installed into `templates/pinax_stripe/*` but some need to be in `templates/pinax/stripe/*` - it's pretty inconsistent. 

Missing template files, too. After install you can go to `localhost:8000/payments/payment-methods/` and Django will raise an error about missing template files, looking for `pinax/stripe/paymentmethod_list.html` or `pinax_stripe/card_list.html` (again, inconsistent naming and directory namespacing). But these files don't exist in the Bootstrap theme repo. 

I like that `pinax-stripe` _just works_ (unlike `dj-stripe`) but the templates being in a different repo, docs not referring to the location of them, and the inconsistency has been driving me insane. 

Are you currently working on this? I'd _really_ appreciate if you were!

**Comment by reillychase** [comment]
Yeah same experience here. I think this the way to copy templates but it didnt work for me

https://github.com/pinax/pinax-stripe/issues/555

I also tried dj-stripe and came here after giving up on it, I love the idea of pinax-stripe and it seems to be still being developed but I dont think theres a single person using it. I just found a bug thats existed since Feb 2016 - unable to delete account. I am just learning Django but hope to contribute to this project if I can get up to speed on it, I think wanting to build a SaaS app with django and stripe would be very in demand now and in the future

**Comment by blueyed** [comment]
> I dont think theres a single person using it

We're using it, but not the templates, but rather for integration with DRF.
And we had to do a lot of work to make it work for us after all.

I think the same applies to (both of) you now regarding the templates.

**Comment by KalobTaulien** [comment]
This has turned out quite disastrous, tbh. When I subscribe to a plan, it creates two subscriptions and doesn't let the user delete one of them. Between this an `dj-stripe`, I'm probably just going to write my own Stripe app. `:'(`

**Comment by reillychase** [comment]
Yes same here, I think I’ve put in about 20hours to get to a basic “working” installation of pinax-stripe. By working I mean installed with its bugs as you described - unable to delete account, unable to cancel a subscription

If you write your own maybe we can work together. I am trying to make a simple saas app. DRF backend with VueJS frontend is what I’m thinking would be best

**Comment by blueyed** [comment]
See also https://github.com/pinax/pinax-stripe/issues/492#issuecomment-343542705.
You might want to ping the commenter.
As for the templates you might want to wait for @paltman to comment on it.

From the traceback of https://github.com/pinax/pinax-stripe/issues/555 I assume that it might be something trivial to fix, but have not and will not investigate.

**Comment by skiler07** [comment]
Me too, also there are a lot of 'blank' sections in the documentation, especially in templating and front-end e.g. how do we get the "source" token from cards and add them to the customer model, it could be much more clear.

**Comment by paltman** [comment]
The templates are in `pinax-templates`, specifically https://github.com/pinax/pinax-templates/tree/master/pinax/templates/templates/pinax/stripe

I think the inconsistency in the template paths you found came from that old theme library which has been deprecated.

I'm pretty sure the templates are all working out of the box as I've tested them through an integration with the [Stripe Starter Project](https://github.com/pinax/pinax-starter-projects/tree/stripe).

Documentation is something that is definitely needed.  I'd love any help with that if anyone is willing.  This is open source and I do my best to keep up with PRs and accept collaboration.  @blueyed has helped a TON on this and I'm super grateful for it. 

Let me know if you need any help or pointers in starting your own library, @KalobTaulien.  Feel free to beg/borrow/steal whatever you want from this one.  I haven't yet seen the duplicate subscription issue and I've personally seen this app running SaaS solutions in dozens of instances.  I know that comment was some months ago now, but would you be able to create an issue with details so that I could try to reproduce the scenario and get a fix in place?  In terms of deleting a subscription, you can [cancel](https://github.com/pinax/pinax-stripe/blob/v4.4.0/pinax/stripe/actions/subscriptions.py#L11) it.  Was that what you were looking for?  There isn't the notion of deleting a subscription in the [Stripe API](https://stripe.com/docs/api#subscriptions).

**Comment by ThunderTrent** [comment]
@paltman Templates don't work out of the box with just 'pinax.stripe' installed. It throws missing template errors. When you add pinax_templates, it then throws bootstrap tag errors.
